imgparse/imgparse.py

```diff
@@ -224,6 +233,7 @@ def get_relative_altitude(image_path, exif_data=None, xmp_data=None, session_alt
     :param exif_data: the exif dictionary for the image (optional to speed up processing)
     :param xmp_data: the XMP data of image, as a string dump of the original XML (optional to speed up processing)
     :param session_alt: enable to extract the session agl altitude instead of xmp agl altitude for Sentera imagery
+    :param use_rlf: enable to, if session_alt is not enabled, use rlf data, if available, for Sentera imagery
```

Comment:
I think you might want to avoid function arguments that override each other if possible. I think it would be slightly better to use a single ` extract_from` argument with options like `session, relative, lrf`, defaulting to `relative`.

Reply:
Addressed